views/offramp/financial.pug

```diff
@@ -24,3 +24,7 @@ block content
     
     a(href='/financial/income' role='button' draggable='false') #{__('Go back')}
 
+    +banner('red')
+        p <strong>#{__('Survey Instructions:')}</strong> #{__('If you are finished, ')} <a href="/offramp/confirmation">#{__('click here')}</a>.
```

Comment:
I think this should be a mixin so that if we want to update it, we don't do it on every page.